fullstack/README.md

```diff
@@ -0,0 +1,64 @@
+This is a coding exercise to help assess candidates looking to join our team at Condé Nast International as FullStack Developers.
```

Comment:
Really minor: I think `FullStack` is usually written as two words?  eg. `Full Stack`

Reply:
👍